pkg/asset/installconfig/installconfig.go

```diff
@@ -124,6 +78,16 @@ func (a *InstallConfig) Generate(parents asset.Parents) error {
 		Data:     data,
 	}
 
+	// Apply the defaults *after* marshaling into yaml so that the defaults
```

Comment:
I am a little concerned. So the asset has 2 values of the install-config. one in `File` and other in `Config`. 
And in the past people have tried to use `Files()` on an asset and decode rather than using the type.

Also, another thing which is more of a personal preference.
getting the install-config with values defaults is a good representation of what the installer has chosen for you and what you can modify for yourself.

Reply:
^^ thoughts @crawford @wking @cgwalters

Reply:
> Can we do any tricks to comment out the defaults but still include them?

This sounds really complicated.  As long as the installer handles default-injection after loading an install-config, I don't really care whether we include defaults in configs we write to disk or not.

Reply:
Can we change this so that the defaults are included? We can provide documentation to show which fields can be safely omitted for users who want to slim down.

Reply:
Sure. I'll make that change right after I put the kids to bed.